lib/core/print.toit

```diff
@@ -46,12 +46,30 @@ print_ object:
 
 // Dumps the string $message on stdout and flushes.
 print_string_on_stdout_ message:
-  #primitive.core.print_string_on_stdout
+  write_string_on_stdout_ message
+  write_string_on_stdout_ "\n"
+
+// Dumps the string $message on stdout without a newline and flushes.
+write_on_stdout_ message:
+  write_string_on_stdout_ message.stringify
+
+// Dumps the string $message on stdout without a newline and flushes.
+write_string_on_stdout_ message:
+  #primitive.core.write_string_on_stdout
```

Comment:
I'd consider passing a parameter to the write-primitives instead to only get one flushing. One print -> one write.

Reply:
done.